crates/node/src/types.rs

```diff
@@ -37,10 +35,8 @@ pub enum CatchupStreamMessage {
 pub struct CatchupRequest {
     pub context_id: calimero_primitives::context::ContextId,
     pub application_id: Option<calimero_primitives::application::ApplicationId>,
-    pub source: Option<calimero_primitives::application::ApplicationSource>,
     pub last_executed_transaction_hash: calimero_primitives::hash::Hash,
     pub batch_size: u8,
-    pub public_key: PublicKey,
```

Comment:
we probably want to keep this one? although, right now, there's no demonstration of private key control, so it's not as useful or verifiable, we can maybe reintroduce it when proper catchup of member list are implemented.

Reply:
Sounds good to me